client/my-sites/upgrades/domain-management/dns/dns-list.jsx

```diff
@@ -38,7 +34,7 @@ const DnsList = React.createClass( {
 				<DnsRecord
 					key={ index }
 					dnsRecord={ dnsRecord }
-					deleteDns={ this.deleteDns }
+					deleteDns={ this.handleDeleteDns }
```

Comment:
I was following @gziolo's suggestion from `p4TIVU-1GQ-p2`:
> Prefix methods that triggers action, that sends data to the server or changes state on the page, with `handle`.

I thought it was a nice suggestion, but I'm open to discussion - maybe this convention did not take or there's an even cooler prefix on the block (like `dealWith` or `takeCareOf` :stuck_out_tongue_winking_eye: ).